src/__tests__/fileSystem.test.ts

```diff
@@ -216,20 +226,20 @@ describe('walkDirectory', () => {
       '/.j1-integration/graph/step-2/entities/4.json': '4',
     });
 
-    const cacheDirectory = '/.j1-integration';
     await pMap(Object.keys(vol.toJSON()), async (sourcePath: string) => {
       await symlink({
-        cacheDirectory,
-        sourcePath: sourcePath.substring(cacheDirectory.length + 1),
-        destinationPath: `index/entities/type/${path.basename(sourcePath)}`,
+        sourcePath: sourcePath.substring(getRootStorageDirectory().length + 1),
+        destinationPath: path.join(
+          'index/entities/type/',
+          path.basename(sourcePath),
+        ),
```

Comment:
I think, sadly, you'll have to join all components to make `path.join()` useful.

```
> path.join('a\b', 'c')
'a\b/c'
```

```suggestion
        destinationPath: path.join('index', 'entities', 'type', path.basename(sourcePath)),
```

Reply:
Fixed. Also fixed a few other instances in the tests. I'll do a sweep later this week to pick up any other unfriendly paths.